ci/logstash_releases.json

```
@@ -3,10 +3,10 @@
     "5.x": "5.6.16",
     "6.x": "6.8.23",
     "7.x": "7.17.10",
-    "8.x": "8.7.1"
+    "8.x": "8.8.0"
   },
   "snapshots": {
     "7.x": "7.17.11-SNAPSHOT",
-    "8.x": "8.8.0-SNAPSHOT"
+    "8.x": "8.8.1-SNAPSHOT"
```

Let's have plugins test against the next minor instead:

```suggestion
    "8.x": "8.9.0-SNAPSHOT"
```